common/crypto/src/hkdf.rs

```diff
@@ -8,6 +8,9 @@ use hkdf::{
     },
     Hkdf,
 };
+use sha2::Sha512;
```

Comment:
nit: if we're using a new hkdf somewhere, we might as well go with the blake3 digest because it's supposedly better than sha

Reply:
blake3::Hasher can't be natively used, misses the CryptoProxy trait, so I'll keep it as is